libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -610,6 +610,16 @@ AP_GPS::first_unconfigured_gps(void) const
 }
 
 void
+AP_GPS::broadcast_first_configuration_failure_reason(void) const {
+    uint8_t unconfigured = first_unconfigured_gps();
+    if(drivers[unconfigured] == NULL) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_INFO, "GPS %d: was not found", unconfigured + 1);
```

Comment:
Ok, I more or less got it, but I'm not sure if "we" refers to we developers or the end users.  I think users would be quite confused with this message if it didn't come from an action they took.